MANIFEST.in

```diff
@@ -0,0 +1,3 @@
+include README.md
```

Comment:
README.md is included by default in new enough setuptools (the one in xenial is too old -- it doesn't support markdown at all afaict).

Reply:
Just to clarify, this is the MANIFEST of charmcraft itself, right? not of the charms that it builds?

Reply:
Yeap.